src/app/pages/send/send-crypto-asset-form/family/stacks/use-stacks-common-send-form.tsx

```diff
@@ -42,13 +42,15 @@ export function useStacksCommonSendForm({
     symbol,
     ...routeState,
   });
+
   async function checkFormValidation(
     values: StacksSendFormValues,
     formikHelpers: FormikHelpers<StacksSendFormValues>
   ) {
-    // Validate and check high fee warning first
-    const formErrors = formikHelpers.validateForm();
-    if (isEmpty(formErrors) && new BigNumber(values.fee).isGreaterThan(HIGH_FEE_AMOUNT_STX)) {
+    const formErrors = await formikHelpers.validateForm();
+
+    if (highFeeWarningContext.isHighFeeWithNoFormErrors(formErrors, values.fee)) {
```

Comment:
I think it would be good to have a well named constant for `highFeeWarningContext.isHighFeeWithNoFormErrors(formErrors, values.fee)` rather than pass it to the `if(`

Reply:
destructured, as per other comment, and looks way better now